3rdparty/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+file(GLOB_RECURSE GTEST_CPPS "gtest/googletest/src/*.cc")
+file(GLOB_RECURSE GTEST_HPPS "gtest/googletest/include/*.h")
+
+include_directories(./gtest/googletest/include)
+include_directories(./gtest/googletest/)
```

Comment:
Have you tried to build it in Windows? I believe it's not so easy. Take a look on my repository.